### DISCUSSION TITLE
Automation Routine Grouping

### DISCUSSION THREAD

**Discussion by gittygitlab** [home-assistant/frontend#11313] [Other feature requests]
Hello,

I would love the ability to group automation routines into groups. For example, grouping all my lighting automations vs the home security alerts or grouping automations based on the room they occur in.

Thank you!

**Comment by frenck** [comment]
You can assign automations to area's

**Comment by carpii** [comment]
I'd like this too, but it's been suggested a lot and there seems to be some resistance to creating groups for automations

An alternative idea might be to allow you to assign an arbitrary tag (simple string) to each automation. 
Lovelace could then let you filter by whatever tags have been defined 

It might resolve any issues of trying to maintain a hierarchy in database, and would be much simpler to develop

**Comment by drax-uk** [comment]
Tags or folders would be awesome. Assigning automation to the area doesn't quite solve my problem. I would like to be able to group automation by the switch, notifications, etc...

**Comment by top4ek** [comment]
Looking forward for same ability, because lot of automatons became problem after some time.
For example HA could use directory automation with lot of yamls, one per automation. And they also could be grouped into sub-directories. 
So every file became automation, every subdir — group. You can easily edit via frontend and by going to file system directly. But it requires, i believe, changes in the core.

**Comment by lindhe** [comment]
I really like the idea of having folders with matching directories in the file system for the YAML files. That will help a lot for drawing the right conclusions when configuring the system!
